snc.sh

```diff
@@ -246,6 +246,13 @@ retry ${OC} replace -f pull-secret.yaml
 # Wait for the cluster again to become stable because of remove of pull secret
 wait_till_cluster_stable
 
+# Delete the pods from MCO forcefully as workaround of
+# https://bugzilla.redhat.com/show_bug.cgi?id=1965992
+retry ${OC} delete pods --all -n openshift-machine-config-operator --grace-period=0
+
+# Wait for the cluster again to become stable because removing pods from MCO
```

Comment:
`Wait for the cluster to become stable again after removing the MCO pods` reads better I think